kyaml/yaml/fns.go

```diff
@@ -205,8 +272,8 @@ func (c FieldClearer) Filter(rn *RNode) (*RNode, error) {
 	return nil, nil
 }
 
-func MatchElement(field, value string) ElementMatcher {
-	return ElementMatcher{FieldName: field, FieldValue: value}
+func MatchElement(field string, value string, keys []string, values []string) ElementMatcher {
```

Comment:
MatchElement is exportable functions. Is that okay to change the passing reference? Will it break its potential callers?

Reply:
The only alternative that I can think of would be to keep all of the exportable functions as is and add new ones that do the same thing except take []string instead of string. Should I try to do that instead?

Reply:
I'd like to suggest to add new methods with more parameters.

Reply:
Thanks Donny, I'm working on making that change now